draft-ietf-quic-tls.md

```diff
@@ -387,6 +392,10 @@ peer in CRYPTO frames. When TLS provides handshake octets to be sent, they are
 appended to the current flow and any packet that includes the CRYPTO frame is
 protected using keys from the corresponding encryption level.
 
+Unlike its operations with TCP, the TLS bytestream is never separately
+encrypted and decrypted. This operation only occurs on the QUIC packet as a
+whole.
```

Comment:
I know what you are saying, but it's equally confusing because it's not clear what the TLS bytestream is.  For QUIC, it's the sequence of TLS handshake messages, but TLS has other concepts that could be considered to be a bytestream: the sequence of TLS records (unencrypted), or the sequence of protected TLS records could very much be considered to be a bytestream.

Maybe the right thing to say here is that "QUIC takes the unprotected content of TLS handshake records as the content of CRYPTO frames.  Encapsulation of these messages in protected TLS records does not happen.  QUIC assembles CRYPTO frames into QUIC packets, which are protected using QUIC packet protection."

But isn't that written down elsewhere?

Reply:
I have spent some time looking through the quic-tls draft and cannot find it. That doesn't mean it's not there, but I'll use your language.

I am really looking for a clear statement that says "turn off TLS handshake encryption, dummy".